test/browser/features/BrowserBasedTests/HappyPath/EUDrivingLicence/F2FEUDrivingLicenceSelectPostOfficeScreen.feature

```diff
@@ -2,7 +2,7 @@
 Feature: Document Selection - EU Driving Licence E2E Journey (Happy Path)
 
 Background:
-    Given Authenticatable Anita is using the system
+    Given A UK Drivers Licence User is using the system
```

Comment:
The feature is `EU Driving Licence E2E Journey` but we're using `A UK Drivers Licence User` just want to sense check that is correct?

Reply:
Good spot, all these tests we're using Authenticatable Anita before with a hardcoded yotiMockId of '0000' so I updated the initial step so we could use different docTypes/yotiMockIds for E2E tests. 

Updated all "Given Authenticatable Anita is using the system" to "A UK Drivers Licence User is using the system" but will update to make sure I'm using the correct docType BDD step (but this doesn't impact any of the browser tests, only e2e)